examples/6.educational_pluto_notebook/simulation-example.jl

```diff
@@ -4,16 +4,6 @@
 using Markdown
 using InteractiveUtils
 
-# This Pluto notebook uses @bind for interactivity. When running this notebook outside of Pluto, the following 'mock version' of @bind gives bound variables a default value (instead of an error).
-macro bind(def, element)
-    quote
-        local iv = try Base.loaded_modules[Base.PkgId(Base.UUID("6e696c72-6542-2067-7265-42206c756150"), "AbstractPlutoDingetjes")].Bonds.initial_value catch; b -> missing; end
-        local el = $(esc(element))
-        global $(esc(def)) = Core.applicable(Base.get, el) ? Base.get(el) : iv(el)
-        el
-    end
-end
-
```

Comment:
The `@bind Ax ...` object should be present in the  example  (with no solution)

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/253/commits/adaa322a7898b1cefc2f2e48124daba339136a7f